lightning/src/chain/channelmonitor.rs

```diff
@@ -521,6 +521,56 @@ impl_writeable_tlv_based_enum_upgradable!(ChannelMonitorUpdateStep,
 	},
 );
 
+/// Details about the balance available for claim once the channel appears on chain.
```

Comment:
Precise by balance you mean `to_local` output + HTLC outputs and it negatively change until _all_ HTLCs
are settled.

Reply:
Tweaked it a bit but mostly just pushed for people to look at `get_claimable_balances` which has more details.